### DISCUSSION TITLE
Docker container

### DISCUSSION THREAD

**Discussion by frezeen** [DamianMorozov/OpenTgResearcher#47] [Show and tell]
i do a docker container for this soft, if someone need it

https://hub.docker.com/r/frezeen/tgdownloader

**Comment by DamianMorozov** [comment]
Thank you so much for this docker image. I will put a link to it in our group for discussion.

**Comment by DamianMorozov** [comment]
Added [Guide to setup the docker version](https://github.com/DamianMorozov/TgDownloader/blob/main/GUIDE-SETUP-DOCKER.md)

**Comment by frezeen** [comment]
updated to latest 0.3.10

**Reply by DamianMorozov** [reply]
Hello.
Can you share the Docker-file for creating container?

**Reply by DamianMorozov** [reply]
Hello everyone. I suggest to take part in testing of docker-version.
https://hub.docker.com/r/damianmorozov/tgdownloaderconsole

**Comment by frezeen** [comment]
sorry i read now only about your request, u need it yet?

**Reply by DamianMorozov** [reply]
It's already completed.

**Comment by frezeen** [comment]
ok perfect ill test it.. thanks damian for your work

**Comment by DamianMorozov** [comment]
The next step at progress - is the making Docker image for TgDownloader-Blazor.

**Comment by DamianMorozov** [comment]
Docker image for TgDownloader-Console
https://hub.docker.com/r/damianmorozov/tgdownloader-console

Docker image for TgDownloader-Blazor
https://hub.docker.com/r/damianmorozov/tgdownloader-blazor
